iOS/LocalPackages/AIChat/Sources/AIChat/Views/ToastView.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  ToastView.swift
```

Comment:
I considered using `ActionMessageView` for this, but given this is a separated package, and `ActionMessageView` is still using .xibs, I’ve decided to create a very simple toast that works only for duck.ai needs. We might even expand this in the future so keeping them separated seems good to me.